### ISSUE TITLE
Kodi 18 RC5 mkv playback issues on Xbox One

### ISSUE THREAD

**Issue by hardzsi** [xbmc/xbmc#15254]
### Describe the bug
Trying to move the progress bar back and forth during playback of .mkv files causes various bugs/glitches (detailed below).
 

## Expected Behavior

Proper handling of .mkv files: when using the progress bar, the video should continue normally within 1-2 seconds.
Also, I tried VLC for Xbox on the same .mkv files, and the video handling (including progress bar) works properly there: this means, that the problem is NOT with the video files, but the player. 

## Actual Behavior

Trying to move the progress bar back and forth during playback of an mkv file causes various bugs/glitches:
- green screen for seconds, whilst the speech/music is audible underneath;
- sometimes the picture restores properly from green screen, but mostly not; 
- a few times the video freezes completely and i should exit to main screen;
- after some tries on various mkv files, KODI freezes on exiting - only Xbox button worked afterwards.

### To Reproduce

1. open an .mkv video file
2. move the progress bar slider more times back and forth

### Debuglog

I don't know, what it is, and how to get it on an Xbox.

### Screenshots 

Unfortunatelly I did not make some (I event don't know, how to do it on an Xbox)

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [ ] Windows
 - [x] Windows UWP

 - Operating system version/name: Xbox OS (latest - maybe November 2018 update)
 - Kodi version: 18-rc5

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
> I don't know, what it is, and how to get it on an Xbox.

That's at least pretty honest ;)

Please read this about how to get the debug log:

https://kodi.wiki/view/Log_file/Easy

And please follow each step. Especially to enable debug logging and to reproduce your issue before!! you grab the log. 

Be aware, that we will close this issue within the next 3 days if there's no reaction from your side. I'll mark this as "invalid" for the time being. That's more a notification for us rather than the real meaning that this bug is not valid. It might be, but we need your logfile to confirm. Otherwise we can't do anything unfortunately.

**Comment by LoneFenris** [comment]
I have a feeling this is the issue posted about on the forum [here](https://forum.kodi.tv/showthread.php?tid=337475). If so, disabling hardware acceleration appears to be a workaround. If OP doesn't provide a log, I think I can.

**Comment by LoneFenris** [comment]
Here's my log. I should note, however, that for me, it is not limited to move files as indicated by the use of a stream in the log.

https://paste.kodi.tv/inonohesex

**Comment by hardzsi** [comment]
Following the description I maganged to install and switch the debug mode on.
Afterwards i tried to reproduce the errors described above with at least 6 different .mkv and 2 .mp4 files. This time i could not crash KODI; actually it seemed to be pretty stable and robust, however after moving the progress bar, I had to wait at least 5-10 seconds, before the video playing returned to normal; in the meantime it showed the green issues (green picture with or without sound in background) what LoneFenris and myself talked about.

As i said before, playing the same video files vith VLC has no seeking problems. 

What i noticed however, that playing Big Buck Bunny (https://peach.blender.org/download/) 1080p in .mp4, the seeking problem was way less noticeable. I don't know, that the differece was because of the container format (.mp4 instead of .mkv) or just more I-frames was put among P and B-frames during compression and that helped to recover after seeking.

From my part, the problem is solved, as I can live with it, that after seeking i should wait even 5-10 seconds for the video playback to recover).

So thanks for the feedback! Now i close this ticket.
